python/py-fypp/Portfile

```diff
@@ -18,9 +18,8 @@ long_description    Fypp is a Python-powered preprocessor. It can be used for an
                     when formulating metaprogramming tasks. It puts strong emphasis on robustness \
                     and on neat integration into developing toolchains.
 homepage            https://github.com/aradi/fypp
-checksums           rmd160  ede50d1312a8669d5adb7a3735d37e8c1971e3f8 \
-                    sha256  b777ae70081c7c82edcf52eefc01a3e8a566b4e82ccd33ca3b2f06faf1d07d36 \
-                    size    68237
+checksums           rmd160  01e0861a62215af3601b3da9a652f1e3c4d8fdab \
+                    sha256  05c20f71dd9a7206ffe2d8688032723f97b8c2984d472ba045819d7d2b513bce \
+                    size    69307
 
 python.versions     39 310 311
```

Comment:
does this support PY312? If so, please add that as well.

Reply:
I am waiting for it to be unbroken on ppc prior to touching it (my patch to be merged).

P. S. If anyone would want to try, it is desirable to make sure `fypp` works with its dependents.

Reply:
so, are you saying that it's only broken for PY312 currently but the other Python subports work (that sounds a bit unlikely, but I guess it's possible)?

Reply:
Sorry, missed the message. No, I do not think it is likely for `py-fypp` to have issues with a specific python, rather just in case.